pydra/engine/core.py

```diff
@@ -608,8 +603,8 @@ def is_workflow(obj):
 
 def is_runnable(graph, obj):
     """Check if a task within a graph is runnable"""
-    if graph.predecessors(obj):
-        for pred in graph.predecessors(obj):
+    if graph.connections_pred[obj.name]:
```

Comment:
use `predecessors`

Reply:
but for sorting graph etc. we don't need  this info. right?

Reply:
i've changed some names, but not all, since I sometimes combine two methods from nx into one like `add_node` and `add_nodes_from` into `add_nodes` that works for one element or for a list